### PR TITLE
Add questions

### PR DESCRIPTION
**Pull request by nico-cf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
...
+
+#### Run tests
+```
+go test -v ./...
+```
+
+---
+
+## Submission
+When you are ready to submit, push all of your changes, and add `nico@connectedfreight.com` as a collaborator to your repo for grading 😊
```
Comment (Woodz) [inline README.md]: @nico-cf are you going to to be the main grader for GoLang homework assignments? If so, should we change this?

File: question-1/question_1_test.go [hunk 2]
```diff
@@ -0,0 +1,206 @@
+package question_1_test
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	. "github.com/connected-freight/be-toke-home-assignment-golang/question-1"
```
Comment (Woodz) [inline question-1/question_1_test.go]: Nit:
```suggestion
	. "github.com/connected-freight/be-take-home-assignment-golang/question-1"
```

File: question-2/question_2_test.go [hunk 3]
```diff
@@ -0,0 +1,91 @@
+package question_2_test
+
+import (
+	"reflect"
+	"testing"
+
+	. "github.com/connected-freight/be-toke-home-assignment-golang/question-2"
```
Comment (Woodz) [inline question-2/question_2_test.go]:
```suggestion
	. "github.com/connected-freight/be-take-home-assignment-golang/question-2"
```

**Review by Woodz — APPROVED** [review]
I know nothing about Golang so can't review the actual code - instead I'd reviewed the general changes

File: question-1/question_1_test.go (hunk 2 above)
Comment (nico-cf) [reply question-1/question_1_test.go]: Thanks @Woodz . Good catch.

File: README.md (hunk 1 above)
Comment (nico-cf) [reply README.md]: Sure, that's fine. I've changed the email address to my Github username.

**Comment by nico-cf** [comment]
> I know nothing about Golang so can't review the actual code - instead I'd reviewed the general changes

Thank you for the review. @Woodz

**Pull request merged**